src/pages/ScoreReport.vue

```diff
@@ -73,7 +45,16 @@
             @filter="onFilter($event)"
             @export-all="exportAll"
             @export-selected="exportSelected"
-          />
+          >
+          <label for="ms-columns" class="view-label">View</label>
```

Comment:
Passing in the class `view-label` works with the slot, but then the label is formatted differently than all of the other drop-down labels. Can we find a way to standardize all of the multi-select labels?